build.psm1

```diff
@@ -1135,88 +1105,89 @@ function Start-PSBootstrap {
     $sudo = if (!$NoSudo) { "sudo" }
 
     try {
-        # Update googletest submodule for linux native cmake
         if ($Environment.IsLinux -or $Environment.IsOSX) {
             try {
+                # Update googletest submodule for linux native cmake
```

Comment:
@TravisEz13 and I talked offline. The `-package` parameter is required only for creating Appx package, and before this change `-package` only takes effect on Unix platforms. If we don't care about Appx, for example in our release build, we can just run `Start-PSBootstrap` for windows.

If we can remove the script about Appx package, then we need to make the following subsequent changes:
1. Add switch parameter `-BuildWindowsNative` to install dependencies for building PSRP on Windows.
2. Install `Wix` when running `Start-PSPackage -Package` on Windows.

@SteveL-MSFT can you please confirm if we can remove `New-AppxPackage`? If so, I will take care of (1) in this PR and @TravisEz13 will take care of (2) in his packaging changes.